main.py

```diff
@@ -421,6 +440,12 @@ def main() -> None:
         metavar="PATH",
         help="path to the 'libjs-test262-runner' binary",
     )
+    parser.add_argument(
+        "-b",
+        "--use-bytecode",
+        action="store_true",
+        help="Use the bytecode interpreter to run the tests",
+    )
```

Comment:
needs an update of the `--help` dump in the readme please :)

Reply:
Sure~